src/types/businessCase.ts

```diff
@@ -25,6 +25,9 @@ export type ProposedBusinessCaseSolution = BusinessCaseSolution & {
 
 // Business Case Form Model
 export type BusinessCaseModel = {
+  id?: string;
+  euaUserId?: string;
+  systemIntake: string;
```

Comment:
I think this should be systemIntakeID

Reply:
Sure, I matched the names from the API. It would probably be a good idea to update it there too.